parsl/executors/taskvine/exec_parsl_function.py

```diff
@@ -78,17 +78,10 @@ def remap_all_files(mapping, fn_args, fn_kwargs):
             remap_location(mapping, maybe_file)
 
 
-def unpack_function(function_info, user_namespace):
-    """ Unpack a function according to its encoding scheme."""
-    return unpack_byte_code_function(function_info, user_namespace)
-
-
-def unpack_byte_code_function(function_info, user_namespace):
-    """ Returns a function object, a default name, positional arguments, and keyword arguments
-    for a function."""
-    from parsl.serialize import unpack_apply_message
-    func, args, kwargs = unpack_apply_message(function_info["byte code"], user_namespace, copy=False)
-    return (func, 'parsl_function_name', args, kwargs)
+def unpack_object(serialized_obj, user_namespace):
```

Comment:
(there are some other places in the parsl serialization code where user_ns is passed around where I think it is unnecessary too - probably historical tangling of deserialisation vs execution)

Reply:
Assuming `user_namespace` is, in fact, not needed, this could be reduced by moving the import out of the function (to the module), then:
```python
def unpack_object(serialized_obj):
    return deserialize(serialized_obj)
```

At which point, there's no need for the extra call in the stack.  Just alias the function:
```python
unpack_object = deserialize
```

Alternatively, could alias at import time (with the other module imports):
```python
from parsl.serialize import deserialize as unpack_object
```

Or even just use the `deserialize` method directly, elsewhere (e.g., in `load_function`).